code/server/validations/authValidation.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Import the module for validation checks
+const { check } = require('express-validator');
+
+// Validation checks for registration of a new user
+exports.signup = [
+    check('firstname').isString().notEmpty(),
```

Comment:
For strings also check they don't exceed a maximum length (according to field size in the database)